walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +123,68 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredWithOptionalAttendeeBlockingEntireDay() {
```

Comment:
Just a fyi: for consistency, we follow the "arrange, act, test" model of writing Java unit tests at Google!

The **arrange** step is for setting up details of the external world specific to the situation under test (e.g. in this case, initializing `events`, `request`).

The **act** step is where you actually make calls to the class/object under test. In this case: `query.query(...)`.

The **assert** step is the place to make assertions on the return values collected.

We usually leave a blank line in between the arrange, act, and test steps.

Reply:
Fixed!